python/GafferSceneUI/SceneInspector.py

```diff
@@ -1509,7 +1509,11 @@ def update( self, targets ) :
 						_Rail( _Rail.Type.Middle )
 
 				if i == 0 or i == ( len( history ) - 1 ) or history[i-1].value != history[i].value :
-					GafferUI.NameLabel( history[i].target.scene.node(), formatter = lambda l : ".".join( x.getName() for x in l ) )
+					GafferUI.NameLabel(
+						history[i].target.scene.node(),
+						formatter = lambda l : ".".join( x.getName() for x in l ),
+						numComponents = self.__distance( history[i].target.scene.node().scriptNode(), history[i].target.scene.node() )
+					)
```

Comment:
When we're deeply nested, do we actually need to go all the way up to the scriptNode?  I guess defining the correct reference point would be tricky otherwise ... there are situations where using the parent of target node as the reference ancestor would be nicer, but I guess this doesn't work in the general case where the history extends back to nodes that are outside the current scope.

Reply:
If we want to do this I think the correct reference point would probably be the common ancestor of the inspected node and the upstream node in the history. But I made this PR after finding that some folks hadn't even realised they were looking at node names, so decided to go with absolute clarity at least for now.